lib/exchange/HL7Server.js

```diff
@@ -22,7 +22,7 @@ class HL7Server extends EventEmitter {
    * @param {Object} [param2]
    * @param {string} [param2.applicationName]
    * @param {string} [param2.facilityName]
-   * @param {string} [param2.defaultEncoding]
+   * @param {string} [param2.encoding]
```

Comment:
I did not understand why the `HL7Client` used `encoding` as this option's name and here in the `HL7Server` it is `defaultEncoding`. Unified both to `encoding`. Note that this change may break some clients, so you might want to add `defaultEncoding` too that would be used for receiving HL7 messages as before. `HL7Server` uses the character encoding specified in `encoding` both when received and sent HL7 messages.

Reply:
@erayhanoglu I noticed you merged this PR already, but please check out this comment.